aspnetcore/toc.md

```diff
@@ -219,6 +219,7 @@
 ### [Publish to Azure](xref:signalr/publish-to-azure-web-app)
 ## [Clients](xref:signalr/clients)
 ### [.NET client](xref:signalr/dotnet-client)
+### [.NET API reference](/dotnet/api/microsoft.aspnetcore.signalr.client?view=aspnetcore-2.1)
```

Comment:
Is it necessary to hard-code 2.1 here?

Would an xref link work -- `(xref:Microsoft.AspNetCore.Signalr.Client)`

Reply:
Hard coding to 2.1 isn't necessary.

 >Would an xref link work

I was just following the Java and JavaScript API reference example. They don't use xref so I assumed it shouldn't be used here as well.

Reply:
I just checked with https://xref.docs.microsoft.com/query?uid=Microsoft.AspNetCore.SignalR.Client and the xref link should work.

The Java link should also work as an xref, as https://xref.docs.microsoft.com/query?uid=com.microsoft.aspnet.signalr checks out too.

JavaScript API ref works differently.

Updating them now isn't necessary, it's a direction we want to go in the future.  Info about the xref links is in an [internal guide doc](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-write-links?branch=master).